src/coreclr/src/System.Private.CoreLib/System.Private.CoreLib.csproj

```diff
@@ -48,6 +43,7 @@
     <NoStdLib>true</NoStdLib>
     <NoCompilerStandardLib>true</NoCompilerStandardLib>
     <SubsystemVersion>6.00</SubsystemVersion>
+    <RuntimeMetadataVersion>v4.0.30319</RuntimeMetadataVersion>
```

Comment:
Yes this can be directly passed to the compiler as of ~15.4 (3 years ago).

[https://github.com/dotnet/roslyn/commit/99ddc7265081746d92232b12ade589e9e349aa9c](https://github.com/dotnet/roslyn/commit/99ddc7265081746d92232b12ade589e9e349aa9c)